src/Input/Silk.NET.Input.Common/Thumbstick.cs

```diff
@@ -5,12 +5,32 @@
 
 namespace Silk.NET.Input.Common
 {
+    /// <summary>
+    /// A thumbstick on an <see cref="IGamepad"/>
```

Comment:
Hmmm, should we use `a` or `an`. One one hand, it could be `a gamepad`. But on the other, it's `a IGamepad` which is grammatically incorrect (Grammarly is shouting at me as we speak). What do you think?

Reply:
I definitely think it should be "an".

Reply:
Alright, let's leave it then :)